bin/commands/suite/import-test.js

```diff
@@ -0,0 +1,19 @@
+const helpers = require('../../helpers')
+
+// TODO: node.js client says imports still support Selenium IDE v1 format, should we remove this?
```

Comment:
I believe the [API supports both .html and .json](https://ghostinspector.com/docs/api/suites/#import-test). We should continue to allow both.

Maybe you're thinking of the Selenium v2 Builder (.json) format? I don't think we ever supported import of those files; just export.